docs: document tui improvements (#3724)
spectrocloud/librarium: docs/docs-content/clusters/edge/site-deployment/site-installation/initial-setup.md

   ![A terminal user interface showing displaying network configuration options](/cluster_edge_site-deployment_installation_initial-setup_tui.webp)

   Check the existing hostname and, optionally, change it to a new one. Use the Tab key or the up and down arrow keys to
   switch between fields. When you make a change, press **Enter** to apply the change.

5. In **Host Network Adapters**, select a network adapter you'd like to configure. By default, the network adapters

>    request an IP automatically from the DHCP server. The CIDR block of an adapter's possible IP address is displayed in

Reviewer: ```suggestion
   request an IP automatically from the Dynamic Host Configuration Protocol (DHCP) server. The CIDR block of an adapter's possible IP address is displayed in
```

Nitpick. I think this is the first occurrence of the term DHCP

>    request an IP automatically from the DHCP server. You can see the CIDR block of an adapter's possible IP address from

Author: ```suggestion
   request an IP automatically from the DHCP server. The CIDR block of an adapter's possible IP address is displayed in
```
